addons/l10n_ch/models/res_bank.py

```diff
@@ -4,24 +4,38 @@
 import re
 
 from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
 from odoo.tools.misc import mod10r
 
 import werkzeug.urls
 
+ISR_SUBSCRIPTION_CODE = {'chf': '01', 'eur': '03'}
+CLEARING = "09000"
+_re_postal = re.compile('^[0-9]{2}-[0-9]{1,6}-[0-9]$')
+
+
 def _is_l10n_ch_postal(account_ref):
-    """ Returns True iff the string account_ref is a valid postal account number,
+    """ Returns True if the string account_ref is a valid postal account number,
```

Comment:
well, I suppose `iff` wasn't a typo but stood for _"if and only iff"_, but anyway... your proposal is clearer.